### DISCUSSION TITLE
Any idea for Notification extension

### DISCUSSION THREAD

**Discussion by elitemobile0301** [expo/expo#21857] [Proposals and Ideas]
Hello Expo Support Team,

I am developing an order delivering app and I have difficulty for notification.

Sometimes customers asked me to show order progressing bar and allow buttons such as accept, reject and others.

I added a screenshot that show notification.

we can implement notifications like this?

Please help me to do this.

Thank you,

elitemobile0301

![2023-03-27 21 43 52](https://user-images.githubusercontent.com/51739383/227956648-5d4aaee1-1ece-48f4-893a-7e6373de35e5.jpg)

**Comment by ajo-alex** [comment]
It's an iOS live activity. You can add these by some native changes. 
https://developer.apple.com/documentation/activitykit/displaying-live-data-with-live-activities
